app/views/mailers/pro_membership_mailer/expiring_membership.text.erb

```diff
@@ -1,6 +1,5 @@
-<% host = "#{ApplicationConfig['APP_PROTOCOL']}#{ApplicationConfig['APP_DOMAIN']}" %>
-<% credits_url = url_for(host: host, controller: :credits, action: :index) %>
-<% pro_membership_url = url_for(host: host, controller: :users, action: :edit, tab: "pro-membership") %>
+<% credits_url = url_for(host: app_url, controller: :credits, action: :index) %>
+<% pro_membership_url = url_for(host: app_url, controller: :users, action: :edit, tab: "pro-membership") %>
```

Comment:
this page in the settings doesn't exist anymore

Reply:
I was still able to open it on localhost though. Does it mean that this mailer template doesn't make sense anymore?

Reply:
You're right! The page is still there, I thought it was deleted when we started taking out features for Pro. Anyway, let's keep it there. We can revisit what to do about pro in the future :D